pkg/types/validation/installconfig_test.go

```diff
@@ -1399,6 +1400,42 @@ func TestValidateInstallConfig(t *testing.T) {
 				return c
 			}(),
 		},
+		{
+			name: "valid baseline capability set",
+			installConfig: func() *types.InstallConfig {
+				c := validInstallConfig()
+				c.Capabilities = &configv1.ClusterVersionCapabilitiesSpec{BaselineCapabilitySet: "v4.11"}
+				return c
+			}(),
+		},
+		{
+			name: "invalid baseline capability set specified",
+			installConfig: func() *types.InstallConfig {
+				c := validInstallConfig()
+				c.Capabilities = &configv1.ClusterVersionCapabilitiesSpec{BaselineCapabilitySet: "notValid"}
+				return c
+			}(),
+			expectedError: `"notValid": unknown baseline capability set specified`,
+		},
+		{
+			name: "valid additional enabled capability specified",
+			installConfig: func() *types.InstallConfig {
+				c := validInstallConfig()
+				c.Capabilities = &configv1.ClusterVersionCapabilitiesSpec{BaselineCapabilitySet: "v4.11",
+					AdditionalEnabledCapabilities: []configv1.ClusterVersionCapability{"openshift-samples"}}
+				return c
+			}(),
+		},
+		{
+			name: "invalid additional enabled capability specified",
+			installConfig: func() *types.InstallConfig {
+				c := validInstallConfig()
+				c.Capabilities = &configv1.ClusterVersionCapabilitiesSpec{BaselineCapabilitySet: "v4.11",
+					AdditionalEnabledCapabilities: []configv1.ClusterVersionCapability{"not-valid"}}
+				return c
+			}(),
+			expectedError: `"not-valid": Unknown capability. Must specify one of the following \(openshift-samples\)`,
```

Comment:
We will add more caps over time, so probably worth future-proofing this regexp to `\(.*openshift-samples.*\)` or some such.

Reply:
Fixed in a10be333d